### PR TITLE
Update install instructions

### LINKED ISSUE #147: Improve Linux installation
The Linus installation section in setup.html is incomplete. Given that the target audience is novice coders, it should be expanded to include all the same sections as Mac and Windows e.g.

- The Unix Shell
- Installing Git
- Installing R
- Installing RStudio

### PR DESCRIPTION
**Pull request by zenalapp**
Changed install instructions to be those used by The Carpentries. Resolves #147.

### CODE REVIEW

File: _includes/install_instructions/videoconferencing.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# from: https://github.com/carpentries/workshop-template/blob/gh-pages/_includes/install_instructions/videoconferencing.html
```
Comment (kelly-sovacool) [inline _includes/install_instructions/videoconferencing.html:1]: If you want this to be an html comment (i.e. not shown on the webpage):

```suggestion
<!-- from: https://github.com/carpentries/workshop-template/blob/gh-pages/_includes/install_instructions/videoconferencing.html -->
```

File: _includes/swc/setup.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# from: https://github.com/carpentries/workshop-template/blob/gh-pages/_includes/swc/setup.html
```
Comment (kelly-sovacool) [inline _includes/swc/setup.html:1]:
```suggestion
<!-- from: https://github.com/carpentries/workshop-template/blob/gh-pages/_includes/swc/setup.html -->
```

File: _includes/swc/setup.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+{% assign curricula = "swc-gapminder|swc-inflammation"  | split: "|" %}
+{% unless curricula contains site.curriculum %}
+{% include warning-curriculum.html %}
+{% endunless %}
+
+{% include install_instructions/shell.html %}
```
Comment (kelly-sovacool) [inline _includes/swc/setup.html:8]: Don't forget to add these files!

**Review by kelly-sovacool — CHANGES_REQUESTED** [review]

File: _includes/swc/setup.html (hunk 3 above)
Comment (zenalapp) [reply _includes/swc/setup.html:5]: Done

**Review by kelly-sovacool — APPROVED** [review]

**Pull request merged**